### ISSUE TITLE
The model

### ISSUE THREAD

**Issue by ghost** [AnatoliyLl/SpringAI#2]
Language models are soooo large and only run on very beefy computers. And are too stupid if they aren't as large as 80GB

**Comment by ghost** [comment]
If we get enough money (with donations) we can buy special AI servers. I will attach them in my garage. There we will be able to run large models (ca. 175 Billion parameters)

**Comment by ghost** [comment]
But they cost around 20000$

**Comment by ghost** [comment]
Or we make a paid plan (5$ per month) and they'll get access to GPT3 via my "proxy" server at home

**Comment by AnatoliyLl** [comment]
> Or we make a paid plan (5$ per month) and they'll get access to GPT3 via my "proxy" server at home

Let's sell Andesaurux for 5000 rubles(50 euro) per copy and SisijaFight(without AI) for 2000 rubles(20 euro)

**Comment by ghost** [comment]
No, I wouldn't sell. SisijaFight paid plan only for the Ai. And andesaurux too

**Comment by ghost** [comment]
I think we Needn't make our own Ai because it has to be clever and smart. Not dumb and stupid
